scripts/install.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+set -e -o pipefail
+
+cd `dirname $0`/..
+
+if [[ "$(node --version)" < "v5" ]]; then
```

Comment:
No, no one, but we were always striving to have node 4.x support, but I guess it is fine to not have it on this project.